docs/docs/docs/01-core/utils/traverse-entity.md

```diff
@@ -0,0 +1,142 @@
+---
+title: traverseEntity
+tags:
+  - utils
+---
+
+
+# Traverse Entity Utility
+
+The `traverseEntity` function is designed to recursively traverse a data entity based on its schema definition. It provides hooks to execute custom logic at each level of the traversal through the `Visitor` function.
+
+---
+
+## Overview
+
+### Purpose
+The utility is used for structured data manipulation. It supports various types of attributes, such as relations, components, dynamic zones, and media, allowing seamless traversal and transformation of deeply nested entities.
+
+---
+
+## Function: `traverseEntity`
+
+### Signature
+```typescript
+async function traverseEntity(
+  visitor: [Visitor](#visitoroptions),
+  options: TraverseOptions,
+  entity: Data
+): Promise<Data>;
+```
+
+### Parameters
+- **`visitor`**:  
+  A function executed for each attribute of the entity. It provides context and utilities for modifying the entity.
+
+- **`options`**:  
+  Configuration for traversal, including:
+  - `schema` *(Model)*: The schema definition of the current entity.
+  - `path` *(Path, optional)*: Tracks the current traversal path.
+  - `parent` *(Parent, optional)*: Information about the parent attribute.
+  - `getModel` *(Function)*: A function to retrieve the schema of a model by its UID.
+
+- **`entity`**:  
+  The data entity to be traversed.
+
+### Returns
+A `Promise<Data>` that resolves with the transformed entity.
+
+---
```

Comment:
I'm curious why you use these separators? If it's for styling purposes I feel like it's better to update the headings' CSS than to add these